geopandas/io/tests/test_file.py

```diff
@@ -216,7 +216,7 @@ def test_to_file_int64(tmpdir, df_points):
 def test_to_file_empty(tmpdir):
     input_empty_df = GeoDataFrame()
```

Comment:
```suggestion
    input_empty_df = GeoDataFrame(geometry=[])
```

This will fix the CI. But we should probably still raise an error for an empty GeoDataFrame without geometry as that fails. Can you look into that?

Reply:
I just had a look at the CI. When trying to write a GeoDataFrame without geometry, you get the pretty descriptive error
```
AttributeError: No geometry data set yet (expected in column 'geometry'.)
```
Hence, I don't think we need to raise an extra error here.